ProcessLib/Output/ProcessOutput.cpp

```diff
@@ -10,14 +10,18 @@
 
 #include "ProcessOutput.h"
 
+#ifndef _WIN32
+#ifndef __APPLE__
+#include <cfenv>
+#endif  // __APPLE__
+#endif  // _WIN32
```

Comment:
What happens on WIN32 and APPLE platforms when floating point exceptions occur?

Reply:
TL;DR; Same as before. Change only for linux.

Old behaviour everywhere but on linux: On windows nothing happens at all. On mac the exceptions are not changed for this particular snippet, i.e. if FPE are set **and** if FPE happens in vtk somewhere, it will stop the programme, as before. See code comments below, where it is used for more explanations.